docs/getting_started.md

```diff
@@ -129,6 +129,9 @@ device = torch.device(device)
 model = init_recognizer(config_file, checkpoint_file, device=device)
 
 # test a single video and show the result:
+# The real path for the video to get in this scripts will be `osp.path(data_prefix, video)`.
```

Comment:
`real path for the video` is vague, because user is not sure what the `video` mean. does it mean the folder name of frames when using rawframe?
it will be good if you provide examples, rather than giving abstract description